lib/plug/conn/cookies.ex

```diff
@@ -92,18 +87,21 @@ defmodule Plug.Conn.Cookies do
   defp pad(number), do: Integer.to_string(number)
 
   defp rfc2822({{year, month, day} = date, {hour, minute, second}}) do
-    weekday_name  = weekday_name(:calendar.day_of_the_week(date))
-    month_name    = month_name(month)
-    padded_day    = pad(day)
-    padded_hour   = pad(hour)
+    weekday_name = weekday_name(:calendar.day_of_the_week(date))
+    month_name = month_name(month)
+    padded_day = pad(day)
+    padded_hour = pad(hour)
     padded_minute = pad(minute)
     padded_second = pad(second)
-    binary_year   = Integer.to_string(year)
-
-    weekday_name <> ", " <> padded_day <>
-      " " <> month_name <> " " <> binary_year <>
-      " " <> padded_hour <> ":" <> padded_minute <>
-      ":" <> padded_second <> " GMT"
+    binary_year = Integer.to_string(year)
+
+    weekday_name <>
+      ", " <>
+      padded_day <>
+      " " <>
+      month_name <>
+      " " <>
+      binary_year <> " " <> padded_hour <> ":" <> padded_minute <> ":" <> padded_second <> " GMT"
```

Comment:
It feels the formatter could do a better job here. I will investigate.

Reply:
Yes, good catch, I noticed this but forgot to point it out. 💟